### ISSUE TITLE
Mobile: Hamburger menu changes

### ISSUE THREAD

**Issue by wibily** [TerriaJS/terriajs#5782]
- make About link the same style
- Add 'Help" option to link to external help page (https://terriajs.gitbook.io/terriajs/)
 ![IMG_2224 PNG](https://user-images.githubusercontent.com/4196561/131801653-16d5c6ad-d720-4dfe-90c4-13e83bfc0752.png)

**Comment by na9da** [comment]
- Fix for about menu item style - #5796 
- Help link, to discuss with @AnaBelgun

**Comment by na9da** [comment]
This needs to go in `config.json` as the last array entry under `parameters > helpContent`

```json
{
        "itemName": "mapuserguide",
        "title": "translate#helpPanel.mapUserGuide",
        "paneMode": "externalLink",
        "url": "https://terriajs.gitbook.io/terriajs/"
   }
```
